README.md

```diff
@@ -1,5 +1,7 @@
 # Status
 
+![NetflixOSS Lifecycle](https://img.shields.io/osslifecycle/indeedeng/status.svg)
```

Comment:
Does this need to say "NetflixOSS"? I know that's not visible except as the <img> alt tag, but still, it confuses me.

Reply:
bump @DuaneOBrien

Reply:
Yeah this was a copy/paste artifact. It's how shields.io references the shield in the code they spit out, and it just got perpetuated. I've updated the outstanding PRs, I'll go through and update them everywhere else (and open an issue with shields.io to fix it from their end as well).